src/app/Shared/Services/NotificationChannel.service.tsx

```diff
@@ -221,8 +221,12 @@ export class NotificationChannel {
   }
 
   private logError(title: string, err: any): void {
-    window.console.error(err.stack);
-    this.notifications.danger(title, JSON.stringify(err.message));
+    window.console.error(err?.message);
+    window.console.error(err?.stack);
+
+    if(err?.message!!) {
```

Comment:
What is the double-`!` used for here? Maybe it was meant to be `!!err?.message` to check the truthiness of the nullable property?

Reply:
Oops, my bad. Should be fixed now